docs/setup_docs.sh

```diff
@@ -53,7 +53,7 @@ cd tmp
 
 # Since there's no documentation yet available for a release branch,
 # we only get the documentation from the main branch
-integrate_connector_docs elasticsearch main
+integrate_connector_docs elasticsearch v3.0.0
```

Comment:
Do we want to integrate the 3.0.0 release with Flink SNAPSHOTS? Or should Flink's `master` point to the Elasticsearch `main` branch while Flink's `release-1.16` points to Elasticsearch's `3.0.0` ?

Reply:
I'm not sure actually. Either way the version in the master docs will not actually exist.

I'd be fine with only making this change in 1.16

Reply:
although it would be better if 1.17 referred to 3.0.0 by default instead of `main`

Reply:
so maybe we should hack some `TBD` special case into the connector_artifact shortcode

Reply:
I think that it's best to point that the docs in master point to a version that doesn't exist. But perhaps we should do the same as we do right now, basically say "Only available for stable releases" ?